### DISCUSSION TITLE
Dataset Format

### DISCUSSION THREAD

**Discussion by kadattack** [bowenc0221/panoptic-deeplab#73] [Q&A] [answered]
I would like to know if this implementation of panoptic segmentation requires annotations of "bounding boxes" in the json file.
For example I used the COCO panoptic dataset 2017 where it has "bbox" information stored under "segments_info" in the json file.
Yet if I remove the the bbox from the file (panoptic_val2017.json), the training still works without any errors.

I am asking because I am trying to create a custom dataset in the format of COCO and would like to know if the bounding boxes are even required for training.

**Comment by bowenc0221** [comment, marked as answer]
Panoptic-DeepLab does not need bounding boxes in the json file, it only uses the segmentations.
